pkg/customers/address.go

```diff
@@ -171,3 +172,38 @@ type updateCustomerAddressRequest struct {
 	address   `json:",inline"`
 	Validated bool `json:"validated"`
 }
+
+func addressTypeToModel(v client.AddressType) (AddressType, error) {
+	v = client.AddressType(strings.ToLower(string(v)))
+	m := map[client.AddressType]AddressType{
+		client.ADDRESSTYPE_PRIMARY:   AddressType_Primary,
+		client.ADDRESSTYPE_SECONDARY: AddressType_Secondary,
+	}
+
+	addrType, ok := m[v]
+	if !ok {
+		return 0, fmt.Errorf("unknown addresss type: '%s'", v)
+	}
+	return addrType, nil
+}
+
+type AddressType int
+
+const (
+	AddressType_Primary   AddressType = 1
```

Comment:
Go linters recommend using pure camelCase here, but for enums I've found using a `_` to separate the type and name really helps readability, especially for long enums.

Compare `AddressTypePrimary` vs `AddressType_Primary`.
or 
`CustomerStatusReceiveOnly` vs `CustomerStatus_ReceiveOnly`

Reply:
Isn't the point of creating a `type AddressType int` to make it so callers reference it as `AddressType.Primary`? Personally I prefer to stick to the standard Go naming conventions.

Reply:
http.StatusSwitchingProtocols, http.StatusNonAuthoritativeInfo :)

Reply:
This type is used to store enums as `int` on the backend. `AddressType.Primary` isn't possible here, you can get that by doing 
```go
var (
	AddressType = struct {
		Primary int
	}{
		Primary: 1,
	}
)
```
but we wouldn't want to do that since `AddressType`  would be mutable. Sadly, Go doesn't have an enum type.

Reply:
Also, I don't mind changing it back to standard camelCase

Reply:
Sure, Go doesn't have enums, but the "norm" is to create them by creating a custom type + const() declarations, which is what you've done. Typically, though, the naming doesn't stutter, so it'd be:
```go
type AddressType int

const (
    Primary AddressType = 1
    ....
)
```

Reply:
Sorry, realized my mistake above. Callers would call `address.Primary`, not `AddressType.Primary`.

Reply:
Ah I see what you're saying. I wouldn't go that route unless this was defined in a small, scoped package. Since it'd look like `customers.Primary` to any outside callers. Or just `Primary` when used in this package; there's no quick way to tell that it's strictly `Primary` for `AddressType`.
Also, you'd end up with naming collisions similar to the issue @adamdecaf had here: https://github.com/moov-io/customers/issues/157#issuecomment-692908361

Reply:
Once again, I read too quickly ;P I looked at the file name and assumed it was the package name.

Reply:
> Sorry, realized my mistake above. Callers would call `address.Primary`, not `AddressType.Primary`.

Ah I see. That could work if we put this in an `address` package, which I wouldn't be opposed to; that could be part of a bigger refactor since we should do that for `address`, `phone`, `metadata`, etc.

Reply:
I'm up to speed now. I jumped to conclusions here. I think sticking to Go's naming conventions is the best option, but you're definitely right about the confusion of it being `customers.Primary` since it's not in its own package.

Reply:
👍🏻 I'll set em to camelCase